src/infrastructure/constants/colors.js

```diff
@@ -1,3 +1,5 @@
+const BASE_BLUE_OPACITY = opacity => `rgba(64, 140, 228, ${opacity})`;
```

Comment:
This is a neat approach. :O Curious if there was a particular rationale for this vs using the existing `LIGHT_BLUE` variable, though.

Reply:
It basically comes down to contrast. There are some shades I like better for this, but they provide poor contrast against text content, especially the links. If we want to recolor the links in highlighted rows, we might have some more options. Here's what I mean (note labels in character column):
![image](https://user-images.githubusercontent.com/29510990/58575066-627d2600-820f-11e9-827c-3ead3ece5c20.png)

Light blue here is too bright for a dark theme, I believe. Link color aside, I had to make the text black for it to readable. Sidebar: it's pretty neat the icons obey css color!

Dark blue looks great as far as the color itself, but suffers from contrast problems, especially the link.

Reply:
Okay, yep, makes sense. Good thinking. :)